editor/editor_feature_profile.h

```diff
@@ -49,7 +49,8 @@ class EditorFeatureProfile : public RefCounted {
 		FEATURE_SCRIPT,
 		FEATURE_ASSET_LIB,
 		FEATURE_SCENE_TREE,
-		FEATURE_NODE_DOCK,
+		FEATURE_SIGNALS_DOCK,
+		FEATURE_GROUPS_DOCK,
```

Comment:
Do not remove the Node dock constant, and add the new two at the end of the enum.